openupgrade_scripts/scripts/hr_holidays/16.0.1.5/post-migration.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Coop IT Easy (https://coopiteasy.be)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def set_allocation_validation_type(env):
+    """Operate like the _compute_allocation_validation_type() function.
+
+    It set "no" by default except if employee_request is set to "no"
+    where it set "officer".
+    """
+    openupgrade.logged_query(
```

Comment:
You only need to reallocate those with `set` value to the corresponding one. Please don't write the rest that are OK.

Reply:
As further explanation, imagine there's an extra module adding a new validation type. You are overwriting the leave types with this new validation type in this SQL.